okta/resource_okta_app_oauth_api_scope.go

```diff
@@ -31,30 +31,29 @@ var validScopes = []string{
 	"okta.policies.manage", "okta.policies.read",
 }
 
-func resourceAppOAuthApiScope() *schema.Resource {
+func resourceAppOAuthAPIScope() *schema.Resource {
```

Comment:
Assuming since this is internal only, changing the func name, this won't cause any BC issues?

Reply:
Changing the names of the function doesn't change the provider's behavior.